scripts/create-clusters.sh

```diff
@@ -107,10 +112,12 @@ function create-clusters() {
     unset KUBECONFIG
   fi
 
-  # TODO(font): Configure insecure registry on kind host cluster. Remove once
-  # https://github.com/kubernetes-sigs/kind/issues/110 is resolved.
-  echo "Configuring insecure container registry on kind host cluster"
-  configure-insecure-registry-on-cluster 1
+  if [[ "${CONFIGURE_INSECURE_REGISTRY}" ]]; then
```

Comment:
I'm not sure I understand the need for this change. Since this flag is intended to control configuring the host for an insecure registry, it needs to be separate from whether or not we need to do it in the cluster. The default has always been to explicitly configure it in the cluster since it's always bootstrapped anew and doesn't hurt even if the user won't use it.

Reply:
I'm leery of having to run code if I don't need to, and I've stopped using a local registry after the recent  breakage.  I'll remove it from this PR and add a flag that supports disabling this behavior explicitly.